lib/rubocop/cop/lint/duplicate_methods.rb

```diff
@@ -0,0 +1,61 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for duplicate methods in classes and modules.
+      #
+      # @example
+      #
+      #   class SomeClass
+      #     def some_method
+      #     end
+      #     def some_method
+      #     end
+      #   end
+      class DuplicateMethods < Cop
+        MSG = 'duplicate methods at lines %s detected.'
+
+        def on_class(node)
+          _, _, body = *node
```

Comment:
We prefer the use of more descriptive names than `_` in most cases - like `_superclass` for instance.